dspace-api/src/main/java/org/dspace/discovery/SolrServiceImpl.java

```diff
@@ -900,8 +901,20 @@ protected SolrQuery resolveToSolrQuery(Context context, DiscoverQuery discoveryQ
         if (0 < discoveryQuery.getHitHighlightingFields().size()) {
             solrQuery.setHighlight(true);
             solrQuery.add(HighlightParams.USE_PHRASE_HIGHLIGHTER, Boolean.TRUE.toString());
+            boolean escapeHTML = configurationService.getBooleanProperty("discovery.highlights.escape-html");
```

Comment:
Please set default to `true`.  For example:
`configurationService.getBooleanProperty("discovery.highlights.escape-html", true);`